CHANGELOG.md

```diff
@@ -11,6 +11,7 @@
 - Added support for `artifact_repository` parameter to `Session.add_packages`, `Session.add_requirements`, `Session.get_packages`, `Session.remove_package`, and `Session.clear_packages`
 - Added `fetch_merge_count` parameter to `DataFrameReader.dbapi` (PrPr) for optimizing performance by merging multiple fetched data into a single Parquet file.
 - Added support for Databricks in `DataFrameReader.dbapi` (PrPr).
+- Added support for ingestion with snowflake UDTF in `DataFrameReader.dbapi` (PrPr).
```

Comment:
just wondering if all dbapi changelogs change be consilidated into one big bullet list

Reply:
I think we can, will do this in another PR if needed